### ISSUE TITLE
Bug: ERROR openvpn: Unrecognized option or missing or extra parameter(s) in /etc/openvpn/target.ovpn:17: data-ciphers-fallback (2.5.4)

### ISSUE THREAD

**Issue by MrJamesT** [qdm12/gluetun#837]
### Is this urgent?

Yes

### Host OS

Unraid

### CPU arch

x86_64

### VPN service provider

Private Internet Access

### What are you using to run the container

docker-compose

### What is the version of Gluetun

Running version latest built on 2022-02-06T20:18:09.362Z (commit 2a19b68)

### What's the problem 🤔

I'm trying to install your package on my new unraid NAS (I've been succesfully using it on my synology NAS). However I can't get it working using the same (a bit modified) docker-compose I used on my old NAS. 

I keep getting
`2022/02/06 23:05:07 ERROR openvpn: Unrecognized option or missing or extra parameter(s) in /etc/openvpn/target.ovpn:17: data-ciphers-fallback (2.5.4)`
when I try to start the package. I've removed most of the options I had in my docker-compose however it still doesn't work.

### Share your logs

```log
2022/02/06 23:05:07 INFO Settings summary:
├── VPN settings:
| ├── VPN provider settings:
| | ├── Name: private internet access
| | ├── Server selection settings:
| | | ├── VPN type: openvpn
| | | ├── Regions: czech republic, romania, austria
| | | └── OpenVPN server selection settings:
| | | ├── Protocol: UDP
| | | └── Private Internet Access encryption preset: strong
| | └── Automatic port forwarding settings:
| | ├── Enabled: yes
| | └── Forwarded port file path: /tmp/gluetun/forwarded_port
| └── OpenVPN settings:
| ├── OpenVPN version: 2.5

| ├── User: [set]
| ├── Password: [set]
| ├── Ciphers: []
| ├── Private Internet Access encryption preset: strong
| ├── Tunnel IPv6: no
| ├── Network interface: tun0
| ├── Run OpenVPN as: root
| └── Verbosity level: 1
├── DNS settings:
| ├── DNS server address to use: 127.0.0.1
| ├── Keep existing nameserver(s): no
| └── DNS over TLS settings:
| ├── Enabled: yes
| ├── Update period: every 24h0m0s
| ├── Unbound settings:
| | ├── Authoritative servers:
| | | └── cloudflare
| | ├── Caching: yes
| | ├── IPv6: no
| | ├── Verbosity level: 1
| | ├── Verbosity details level: 0
| | ├── Validation log level: 0
| | ├── System user: root
| | └── Allowed networks:
| | ├── 0.0.0.0/0
| | └── ::/0
| └── DNS filtering settings:
| ├── Block malicious: yes
| ├── Block ads: no
| ├── Block surveillance: no
| └── Blocked IP networks:
| ├── 127.0.0.1/8
| ├── 10.0.0.0/8
| ├── 172.16.0.0/12
| ├── 192.168.0.0/16
| ├── 169.254.0.0/16
| ├── ::1/128
| ├── fc00::/7
| ├── fe80::/10
| ├── ::ffff:7f00:1/104
| ├── ::ffff:a00:0/104
| ├── ::ffff:a9fe:0/112
| ├── ::ffff:ac10:0/108
| └── ::ffff:c0a8:0/112
├── Firewall settings:
| └── Enabled: yes
├── Log settings:
| └── Log level: INFO
├── Health settings:
| ├── Server listening address: 127.0.0.1:9999
| ├── Address to ping: github.com
| └── VPN wait durations:
| ├── Initial duration: 5s
| └── Additional duration: 5s
├── Shadowsocks server settings:
| └── Enabled: no
├── HTTP proxy settings:
| └── Enabled: no
├── Control server settings:
| ├── Listening address: :8000
| └── Logging: yes
├── OS Alpine settings:
| ├── Process UID: 99
| ├── Process GID: 100
| └── Timezone: Europe/Prague
├── Public IP settings:
| ├── Fetching: every 12h0m0s
| └── IP file path: /tmp/gluetun/ip
└── Version settings:

└── Enabled: yes
2022/02/06 23:05:07 INFO routing: default route found: interface eth0, gateway 172.18.0.1
2022/02/06 23:05:07 INFO routing: local ethernet link found: gretap0
2022/02/06 23:05:07 INFO routing: local ethernet link found: erspan0
2022/02/06 23:05:07 INFO routing: local ethernet link found: eth0
2022/02/06 23:05:07 INFO routing: local ipnet found: 172.18.0.0/16
2022/02/06 23:05:07 INFO routing: default route found: interface eth0, gateway 172.18.0.1
2022/02/06 23:05:07 INFO routing: adding route for 0.0.0.0/0
2022/02/06 23:05:07 INFO firewall: firewall disabled, only updating allowed subnets internal list
2022/02/06 23:05:07 INFO routing: default route found: interface eth0, gateway 172.18.0.1
2022/02/06 23:05:07 INFO TUN device is not available: open /dev/net/tun: no such file or directory; creating it...
2022/02/06 23:05:07 INFO firewall: enabling...
2022/02/06 23:05:07 INFO firewall: enabled successfully
2022/02/06 23:05:07 INFO pprof http server listening on [::]:6060
2022/02/06 23:05:07 INFO dns over tls: using plaintext DNS at address 1.1.1.1
2022/02/06 23:05:07 INFO http server: listening on :8000
2022/02/06 23:05:07 INFO healthcheck: listening on 127.0.0.1:9999
2022/02/06 23:05:07 INFO firewall: setting VPN connection through firewall...
2022/02/06 23:05:07 ERROR openvpn: Unrecognized option or missing or extra parameter(s) in /etc/openvpn/target.ovpn:17: data-ciphers-fallback (2.5.4)

2022/02/06 23:05:07 INFO openvpn: Use --help for more information.
2022/02/06 23:05:07 ERROR vpn: exit status 1

2022/02/06 23:05:07 INFO vpn: retrying in 15s
2022/02/06 23:05:12 INFO healthcheck: program has been unhealthy for 5s: restarting VPN

2022/02/06 23:05:20 WARN Caught OS signal, shutting down
2022/02/06 23:05:20 ERROR http server: http: Server closed
```

### Share your configuration

```yml
gluetun:
    container_name: gluetun
    image: qmcgaw/gluetun
    cap_add:
      - NET_ADMIN
    restart: unless-stopped
    volumes:
      - /mnt/user/appdata/gluetun:/gluetun
    environment:
      - PUID=99
      - PGID=100
      - TZ=Europe/Prague
      - VPNSP=private internet access
      - OPENVPN_USER=***
      - 'OPENVPN_PASSWORD=******'
      - REGION=Czech Republic,Romania,Austria
      - PORT_FORWARDING=on
    ports:
      - 8500:8500
```

**Comment by smtm50** [comment]
Getting same error after updating my docker image. App status is unhealthy. 

2022/02/06 17:21:15 INFO routing: default route found: interface eth0, gateway 172.19.0.1
2022/02/06 17:21:15 INFO routing: local ethernet link found: eth0
2022/02/06 17:21:15 INFO routing: local ipnet found: 172.19.0.0/16
2022/02/06 17:21:15 INFO routing: default route found: interface eth0, gateway 172.19.0.1
2022/02/06 17:21:15 INFO routing: adding route for 0.0.0.0/0
2022/02/06 17:21:15 INFO firewall: firewall disabled, only updating allowed subnets internal list
2022/02/06 17:21:15 INFO routing: default route found: interface eth0, gateway 172.19.0.1
2022/02/06 17:21:15 INFO TUN device is not available: open /dev/net/tun: no such file or directory; creating it...
2022/02/06 17:21:15 INFO firewall: enabling...
2022/02/06 17:21:15 INFO firewall: enabled successfully
2022/02/06 17:21:15 INFO pprof http server listening on [::]:6060
2022/02/06 17:21:15 INFO dns over tls: using plaintext DNS at address 1.1.1.1
2022/02/06 17:21:15 INFO http server: listening on :8000
2022/02/06 17:21:15 INFO healthcheck: listening on 127.0.0.1:9999
2022/02/06 17:21:15 INFO firewall: setting VPN connection through firewall...
2022/02/06 17:21:15 ERROR openvpn: Unrecognized option or missing or extra parameter(s) in /etc/openvpn/target.ovpn:18: data-ciphers-fallback (2.5.4)
2022/02/06 17:21:15 INFO openvpn: Use --help for more information.
2022/02/06 17:21:15 ERROR vpn: exit status 1
2022/02/06 17:21:15 INFO vpn: retrying in 15s
2022/02/06 17:21:20 INFO healthcheck: program has been unhealthy for 5s: restarting VPN
2022/02/06 17:21:30 INFO healthcheck: program has been unhealthy for 10s: restarting VPN
2022/02/06 17:21:30 INFO firewall: setting VPN connection through firewall...
2022/02/06 17:21:30 ERROR openvpn: Unrecognized option or missing or extra parameter(s) in /etc/openvpn/target.ovpn:18: data-ciphers-fallback (2.5.4)
2022/02/06 17:21:30 INFO openvpn: Use --help for more information.
2022/02/06 17:21:30 ERROR vpn: exit status 1
2022/02/06 17:21:30 INFO vpn: retrying in 30s
2022/02/06 17:21:45 INFO healthcheck: program has been unhealthy for 15s: restarting VPN
2022/02/06 17:22:00 INFO firewall: setting VPN connection through firewall...
2022/02/06 17:22:00 ERROR openvpn: Unrecognized option or missing or extra parameter(s) in /etc/openvpn/target.ovpn:18: data-ciphers-fallback (2.5.4)
2022/02/06 17:22:00 INFO openvpn: Use --help for more information.
2022/02/06 17:22:00 ERROR vpn: exit status 1
2022/02/06 17:22:00 INFO vpn: retrying in 1m0s
2022/02/06 17:22:05 INFO healthcheck: program has been unhealthy for 20s: restarting VPN
2022/02/06 17:22:30 INFO healthcheck: program has been unhealthy for 25s: restarting VPN
2022/02/06 17:23:00 INFO healthcheck: program has been unhealthy for 30s: restarting VPN
2022/02/06 17:23:00 INFO firewall: setting VPN connection through firewall...
2022/02/06 17:23:00 ERROR openvpn: Unrecognized option or missing or extra parameter(s) in /etc/openvpn/target.ovpn:18: data-ciphers-fallback (2.5.4)
2022/02/06 17:23:00 INFO openvpn: Use --help for more information.
2022/02/06 17:23:00 ERROR vpn: exit status 1
2022/02/06 17:23:00 INFO vpn: retrying in 2m0s
2022/02/06 17:23:35 INFO healthcheck: program has been unhealthy for 35s: restarting VPN
2022/02/06 17:24:15 INFO healthcheck: program has been unhealthy for 40s: restarting VPN

**Comment by qdm12** [comment]
Thanks for reporting the issue. My bad I only tested it with Wireguard earlier today, it's fixed now in ff3f84f9fd1c6ee7de3deed5883e7fd6ca645c09 on the latest image.

**Comment by smtm50** [comment]
As always thanks for quick support @qdm12
